CTP/ha_repl/src/com/navercorp/cubridqa/ha_repl/Test.java

```diff
@@ -848,8 +851,11 @@ private boolean __isDatabaseDirty() throws Exception {
 		s.append("select 'TABLE'||':db_meth_file', t.* from db_meth_file t; ");
 		s.append("select 'TABLE'||':db_serial', t.* from db_serial t; ");
 		s.append("select 'TABLE'||':db_user', t.* from db_user t where name not in ('DBA', 'PUBLIC'); ");
-
-		GeneralScriptInput script = new GeneralScriptInput("csql -u dba " + hostManager.getTestDb() + " -c \"" + s.toString() + "\"");
+ 
+		String spt = "cd $CUBRID;";
+		spt += "csql -u dba " + hostManager.getTestDb() + " -c \"" + s.toString() + "\""; 
+		 
+		GeneralScriptInput script = new GeneralScriptInput(spt);
```

Comment:
Below is general way:

```suggestion
		GeneralScriptInput script = new GeneralScriptInput("cd $CUBRID");
		script.addCommand("csql -u dba " + hostManager.getTestDb() + " -c \"" + s.toString() + "\"");
```